packages/bpk-component-select/src/bpk-select.scss

```diff
@@ -59,6 +59,10 @@ $select-image-large-height: 22 * $bpk-one-pixel-rem;
   flex-direction: row;
   align-items: center;
   background: none;
+
+  @include bpk-rtl {
+    padding-left: 0;
```

Comment:
You should also have to reset the margin-right you previously set i.e:

@include bpk-rtl {
  padding-right: ${some value};
  padding-left: 0;
}